scrapi/base/__init__.py

```diff
@@ -89,7 +89,8 @@ class JSONHarvester(BaseHarvester, JSONTransformer):
     def normalize(self, raw_doc):
         transformed = self.transform(json.loads(raw_doc['doc']), fail=settings.RAISE_IN_TRANSFORMER)
         transformed['shareProperties'] = {
-            'source': self.short_name
+            'source': self.short_name,
+            'docID': raw_doc['docID']
```

Comment:
do we want to stick filetype in here as well, just in case anyone ever wants to parse it out?